src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -20,79 +21,99 @@ tags:
 
 ### PageSpeed Insights (PSI)
 
-[PSI](https://pagespeed.web.dev/) is a tool that reports field data from CrUX _and_ field lab from Lighthouse for a given page.
+[PSI](https://pagespeed.web.dev/) is a tool that reports field data from CrUX _and_ lab from Lighthouse for a given page. See those individual sections for more details.
 
 #### When to use PSI
 
-PSI is great for assessing performance at the page level. It's a good choice for those in non-developer roles who wish to assess how specific pages are performing in the field, which is valuable if you're not gathering your own field data. PSI also provides recommendations for non-developers who are not already using Lighthouse, and from a more consistent baseline than running Lighthouse from your own laptop.
+PSI is great for assessing CrUX performance at the page level or origin-level, for both mobile and desktop users. It's a good choice for an initial overview of Core Web Vitals for a page or site. It also allows you to easily view Core Web Vitals data for other sites like competitors.
+
+As PSI also provides Lighthouse data, which gives useful recommendations to improve your Core Web Vitals—if the metrics  align. Where these do not align, Lighthouse recommendations may be less relevant.
+
+Since Lighthouse is run from the server, it can form a more consistent baseline than running Lighthouse from DevTools.
 
 #### When _not_ to use PSI
 
-PSI may not be worth using for developers already using Lighthouse. From the field data perspective, it's not a fit for websites that aren't represented in the CrUX dataset.
+PSI is only available for public URLs. It cannot be used on development sites that are not publicly accessible.
+
+CrUX data is only available when sites meet certain [eligibility criteria](https://developer.chrome.com/docs/crux/methodology/#eligibility), including site popularity thresholds. PSI is less useful when CrUX data is not available for a page or origin as it can only show the Lighthouse lab data in these cases.
+
+Similarly, if you only have origin-level CrUX data rather than the specific URL being tested, then this also limits its usefulness of correlating the origin-level field data to the page-level lab diagnostics. Having the origin-level field data is still very useful information to have as a summary of the site's performance and the Lighthouse audits may help, but extra caution should be used in this case.
+
+Finally, when page-level data is available in CrUX but this differs from the Lighthouse lab data, then this indicates that the recommendations may be of limited value. This can happen particularly for [post-load CLS issues](/optimize-cls/#identifying-post-load-cls-issues) and for interactivity Core Web Vitals (FID and INP) where lab-based auditing tooling is less useful.
```

Comment:
```suggestion
Finally, where page-level data is available in CrUX, but differs from the Lighthouse lab data, recommendations from Lighthouse may be of limited value. This can happen particularly for [post-load CLS issues](/optimize-cls/#identifying-post-load-cls-issues), and for interactivity Core Web Vitals (FID and INP) where lab-based audits are less useful.
```